File: 1.2 First Year Second Semester/ICT 1204 - Object Oriented Programming Lab/Code/Finallab/src/finallab/employee.java
package finallab;

import java.util.Scanner;

public class employee {

    String name, designation;
    int age, salary;

    public void getData() {

        System.out.println("Detail of a employee:");
        System.out.println("-------------------------");

        System.out.println("Name : " + name);
        System.out.println("Age: " + age);

        System.out.println("Designation: " + designation);
        System.out.println("Salary: " + salary);

    }

    void putData() {
        System.out.println("Detail of a employee:");
        System.out.println("-------------------------");

        Scanner input = new Scanner(System.in);
        System.out.print("Enter Name : ");
        String na = input.nextLine();
        System.out.println("Enter Three numbers : ");
        int age = input.nextInt();
        int designation = input.nextInt();
        int salary = input.nextInt();

    }

   public static void main(String[] args) {
      
        Scanner input = new Scanner(System.in);

        int age = input.nextInt();
        employee[] arr = null;
         for (int i = 0; i < 3; i++) {
           arr[i]=new employee();
         }
    employee na = new employee();
        for (int i = 0; i < 3; i++) {
           arr[i].getData(); 
        }
        for (int i = 0; i < 3; i++) {
            arr[i].putData();
        }
    }
}

File: 1.2 First Year Second Semester/ICT 1204 - Object Oriented Programming Lab/Final Exam/Code/employee.cpp
#include<iostream>

using namespace std;

class Employee
{
    string name;
    int age;
    string designation;
    int salary;
public:
    Employee()
    {
        string temp;
        cout<<"Enter detail of employee:"<<endl;
        cout<<"Name: ";
        getline(cin,name);
        cout<<"Age: ";
        cin>>age;
        cout<<"Designation: ";
        cin>>designation;
        cout<<"Salary: ";
        cin>>salary;
        getline(cin,temp);

        cout<<"Detail of employee:"<<endl;
        cout<<"Name: "<<name<<endl;
        cout<<"Age: "<<age<<endl;
        cout<<"Designation: "<<designation<<endl;
        cout<<"Salary: "<<salary<<endl;
    }
};

int main()
{
    for(int i = 0; i < 2; i++)
        Employee a;

}
